features/bevragen/fields-fout-cases-gba.feature

```diff
@@ -280,152 +280,152 @@ Rule: De fields parameter bevat verkorte/volledig veld paden die verwijzen naar
+    | overlijden.inOnderzoek.datumIngangOnderzoek.onbekend         |
+    | overlijden.inOnderzoek.datumIngangOnderzoek.jaar             |
+    | overlijden.inOnderzoek.datumIngangOnderzoek.maand            |
+    | overlijden.inOnderzoek.datum                                 |
+    | overlijden.inOnderzoek.land                                  |
+    | overlijden.inOnderzoek.plaats                                |
+    | partners.inOnderzoek                                         |
+    | partners.inOnderzoek.datumIngangOnderzoek                    |
+    | partners.inOnderzoek.datumIngangOnderzoek.langFormaat        |
+    | partners.inOnderzoek.datumIngangOnderzoek.type               |
+    | partners.inOnderzoek.datumIngangOnderzoek.datum              |
+    | partners.inOnderzoek.datumIngangOnderzoek.onbekend           |
+    | partners.inOnderzoek.datumIngangOnderzoek.jaar               |
+    | partners.inOnderzoek.datumIngangOnderzoek.maand              |
+    | partners.inOnderzoek.burgerservicenummer                     |
+    | partners.inOnderzoek.geslacht                                |
+    | partners.inOnderzoek.soortVerbintenis                        |
+    | verblijfplaats.inOnderzoek                                   |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek              |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.langFormaat  |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.type         |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.datum        |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.onbekend     |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.jaar         |
+    | verblijfplaats.inOnderzoek.datumIngangOnderzoek.maand        |
+    | verblijfplaats.inOnderzoek.datumIngangGeldigheid             |
+    | verblijfplaats.inOnderzoek.datumVan                          |
+    | verblijfplaats.inOnderzoek.type                              |
+    | verblijfplaats.inOnderzoek.adresseerbaarObjectIdentificatie  |
+    | verblijfplaats.inOnderzoek.functieAdres                      |
+    | verblijfplaats.inOnderzoek.nummeraanduidingIdentificatie     |
+    | verblijfstitel.inOnderzoek                                   |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek              |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.langFormaat  |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.type         |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.datum        |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.onbekend     |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.jaar         |
+    | verblijfstitel.inOnderzoek.datumIngangOnderzoek.maand        |
+    | verblijfstitel.inOnderzoek.aanduiding                        |
+    | verblijfstitel.inOnderzoek.datumEinde                        |
+    | verblijfstitel.inOnderzoek.datumIngang                       |
```

Comment:
moet hier niet ook indicatieVastgesteldVerblijftNietOpAdres aan toegevoegd worden?

Reply:
Er twee abstract scenarios (zie regel 264 en 431) zijn waarmee wordt gevalideerd dat het vragen van automatisch geleverde velden niet mag. Enig verschil is de reason ('Parameter bevat een niet toegestane veldnaam.' en 'Parameter bevat een niet bestaande veldnaam.'). Aan beide scenarios het veld toevoegen zou betekenen dat de GBA API twee verschillende reasons moet teruggeven.